### ISSUE TITLE
Triple click in TextBox does not select all text

### ISSUE THREAD

**Issue by batzen** [dotnet/wpf#1162]
* .NET Core Version: All
* Windows version: 1903
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes

 **Problem description:**
Triple clicking in a `TextBox` does not select all text in the `TextBox`.
 
 **Actual behavior:**
Triple clicking in a `TextBox` does not select all text in the `TextBox`.
 
 **Expected behavior:**
Triple clicking in a `TextBox` should select all text.
 
 **Minimal repro:**
- Create a window containing a `TextBox`
- Write some text into the `TextBox`
- Triple click with the mouse

**Comment by lindexi** [comment]
I do not think it a good feature that it change the behavior. Because I use **Triple clicking in a TextBox does not select all text in the TextBox.** feature to write my bussiness code.  o(╥﹏╥)o 

If your code be merged, I should change my exist code...

**Comment by weltkante** [comment]
Sooner or later WPF needs to add new input gestures and there is always the risk that some application already was using a previously unused input gesture for its own purpose. I think its a good start to think about how to deal with that here, while triple click is not a standard Windows gesture a lot of applications make use of it to select all text. Maybe it can be implemented and hidden behind a switch.

**Comment by mdtauk** [comment]
Triple clicking is the expected behaviour in Windows 10, as used by UWP XAML.
Win32's Edit control only has single click to focus, and double click to select the word the cursor is over.

**Comment by miloush** [comment]
On the other hand, triple click in Word selects the paragraph, not all text.

I would prefer the controls to be consistent with native controls, and I think this case in particular might be quite impactful compatibility issue.

**Comment by batzen** [comment]
It behaves that way for a rich textbox. The change i made does not change the behavior for controls with rich content.

**Comment by batzen** [comment]
@grubioe any news on this?

**Comment by grubioe** [comment]
@fabiant3  can provide the latest status.
